specification/azsadmin/resource-manager/compute/Microsoft.Compute.Admin/preview/2018-07-30-preview/Disks.json

```diff
@@ -0,0 +1,319 @@
+          },
+          {
+            "$ref": "Disks.json#/parameters/TenantSubscriptionIdParameter"
+          },
+          {
+            "$ref": "Disks.json#/parameters/DiskStateParameter"
+          },
+          {
+            "$ref": "Disks.json#/parameters/SourceSharePathParameter"
+          },
+          {
+            "$ref": "Disks.json#/parameters/DiskCountParameter"
+          },
+          {
+            "$ref": "Disks.json#/parameters/DiskStartParameter"
+          },
+          {
+            "$ref": "Disks.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- The list of disks has been returned.",
+            "schema": {
+              "$ref": "Disks.json#/definitions/DiskList"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Compute.Admin/locations/{location}/disks/{DiskId}": {
+      "get": {
+        "x-ms-examples": {
+          "Returns the requested disk.": {
+            "$ref": "./examples/Disks/Get.json"
+          }
+        },
+        "tags": [
+          "Disks"
+        ],
+        "operationId": "Disks_Get",
+        "description": "Returns the disk.",
+        "parameters": [
+          {
+            "$ref": "../2015-12-01-preview/Compute.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../2015-12-01-preview/Compute.json#/parameters/LocationParameter"
+          },
+          {
+            "$ref": "Disks.json#/parameters/DiskIdParameter"
+          },
+          {
+            "$ref": "Disks.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- The disk has been returned.",
+            "schema": {
+              "$ref": "Disks.json#/definitions/Disk"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "DiskProperties": {
+      "description": "Managed disk properties.",
+      "type": "object",
+      "properties": {
+        "diskId": {
+          "description": "The disk id.",
+          "type": "string",
+          "readOnly": false
+        },
+        "status": {
+          "description": "The disk status.",
+          "$ref": "Disks.json#/definitions/DiskState",
+          "readOnly": false
+        },
+        "sharePath": {
+          "description": "The disk share path.",
+          "type": "string",
+          "readOnly": false
+        },
+        "actualSize": {
+          "description": "The actual size of disk.",
+          "type": "integer",
+          "format": "int64",
+          "readOnly": false
+        },
+        "managedBy": {
+          "description": "Compute resource Uri which owns this disk.",
+          "type": "string",
+          "readOnly": false
+        },
+        "tenantId": {
```

Comment:
tenant is the terms we use to describe directory tenants. Everywhere in the docs we say Admin/User (ARM, portal, etc.)

Reply:
Sure, Updated to userResourceId